src/util/storageOptions.tsx

```diff
@@ -7,10 +7,11 @@ export const lvmDriver = "lvm";
 export const zfsDriver = "zfs";
 export const cephDriver = "ceph";
 export const cephFSDriver = "cephfs";
+export const cephObject = "cephobject";
 export const powerFlex = "powerflex";
 export const pureStorage = "pure";
 
-const storageDriverLabels: { [key: string]: string } = {
+export const storageDriverLabels: { [key: string]: string } = {
```

Comment:
```suggestion
const storageDriverLabels: { [key: string]: string } = {
```
I think this was just a work in progress, this is not needed.